src/js/firstparties/tumblr.js

```diff
@@ -0,0 +1,36 @@
+// only observed links with this format out in the wild
+let tumblr_links = "a[href^='https://t.umblr.com/redirect?']";
```

Comment:
This file is almost identical to https://github.com/EFForg/privacybadger/blob/master/src/js/firstparties/google-static.js. What if we reuse `google-static.js`, conditionally setting the DOM selector variable (based on `document.domain` maybe)?